### DISCUSSION TITLE
Add System Services to Sandbox SDK

### DISCUSSION THREAD

**Discussion by phazonoverload** [directus/directus#24694] [Feature Requests]
Add new scopes for key services in the Sandbox SDK:

- [ ] Items
- [ ] Collections
- [ ] Fields
- [ ] Relations
- [ ] Files
- [ ] Users
- [ ] Roles
- [ ] Permissions
- [ ] Policies (v11)

Support only multiple methods (`readItems`, `createUsers`, etc) and not singular methods (`readItem`, `createUser`).

Use the syntax in the public JS SDK and not the existing extension SDK (tradeoff decision between new extensions and migrating existing ones). Must support the same query object syntax as the JS SDK.

```[tasklist]
### Tasks
- [ ] https://github.com/directus/directus/issues/22984
```

**Comment by br41nslug** [comment]
Should we model the Sandbox SDK more after the existing `ItemsServices` instead of the JS SDK? 🤔 Im afraid that if the "backend" (talks directly to the DB) SDK and "frontend" (uses the public REST API) SDK look a lot like each other that it could be confusing to know in which context you're working and what features are available in said context.

**Comment by phazonoverload** [comment]
That decision was made by @alexchopin and I to make it easier for devs who know Directus to move on and build extensions.
